database/seeders/AccountOfficerBranchSeeder.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+namespace Database\Seeders;
+
+use App\Models\User;
+use App\Models\AccountOfficer;
+use Illuminate\Support\Carbon;
+use Illuminate\Database\Seeder;
+use App\Models\AccountOfficerBranch;
+use Illuminate\Support\Facades\DB;
+
+class AccountOfficerBranchSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $account_officer_branch = [
```

Comment:
dili ni pwede i seed gen. kay kani nga data is for production. and dili ni pang dev/admin nga data